iotedgedev/azurecli.py

```diff
@@ -452,7 +452,7 @@ def get_iothub_connection_string(self, value, resource_group):
             if result:
                 out_string = io.getvalue()
                 data = json.loads(out_string)
-                return data["cs"]
+                return data["connectionString"]
```

Comment:
Could we check both `cs` and `connectionString`. Then returns the one that exists? In this way we can support both old and new Azure CLI versions.